cscs-checks/prgenv/cuda/cuda_aware_mpi.py

```diff
@@ -32,79 +32,84 @@ def __init__(self):
 
         self.sanity_patterns = sn.assert_found(r'Finished building'
                                                r' CUDA samples', self.stdout)
-        self.num_tasks = 2
         if self.current_system.name in ['arolla', 'tsa', 'ault']:
             self.exclusive_access = True
             nvidia_sm = '70'
         else:
             nvidia_sm = '60'
 
-        if self.current_system.name in ['daint']:
-            self.prebuild_cmds = ['export CUDA_HOME=$CUDATOOLKIT_HOME']
-        else:
-            self.prebuild_cmds = []
-
         self.prebuild_cmds += ['cd posts/cuda-aware-mpi-example/src']
         gcd_flgs = '-gencode arch=compute_{0},code=sm_{0}'.format(nvidia_sm)
         self.build_system = 'Make'
         self.build_system.options = ['CUDA_INSTALL_PATH=$CUDA_HOME',
                                      'MPI_HOME=$CRAY_MPICH_PREFIX',
                                      'GENCODE_FLAGS="%s"' % (gcd_flgs)]
 
-        self.postbuild_cmds= ['ls ../bin']
-        self.sanity_patterns= sn.assert_found(r'jacobi_cuda_aware_mpi',
+        self.postbuild_cmds = ['ls ../bin']
+        self.sanity_patterns = sn.assert_found(r'jacobi_cuda_aware_mpi',
                                                self.stdout)
-        self.maintainers=['JO']
-        self.tags={'production', 'external_resources'}
+        self.maintainers = ['JO']
+        self.tags = {'production', 'external_resources'}
```

Comment:
```suggestion
        self.tags = {'production', 'scs'}
```

You're not using our external resources anymore.